scripts/bulk-index.js

```diff
@@ -613,7 +623,7 @@ const run = async () => {
   if (
     argv.bibId ||
     argv.itemId ||
-    (argv.type && argv.hasMarc)
+    (argv.type)
```

Comment:
Does this leave the script open to an update on all of a given type? I feel like we should encourage a practice of narrowing down slightly the scope... maybe a check like `argv.type && (argv.hasMarc || argv.nyplSource)`

Reply:
Will implement